ringbuf/reader.go

```diff
@@ -195,3 +195,11 @@ func (r *Reader) BufferSize() int {
 func (r *Reader) Flush() error {
 	return r.poller.Flush()
 }
+
+// AvailableData returns the amount of data available to read in the ring buffer in bytes.
+// This is racy during ongoing operations but is still useful for overall information on how a ringbuffer is behaving.
```

Comment:
To me this line raises more questions than it answers. "racy" implies it's somehow not safe to call? I'd probably just drop the line.